src/lighteval/tasks/templates/formulation.py

```diff
@@ -0,0 +1,56 @@
+from dataclasses import dataclass
+from typing import Literal
+
+from lighteval.tasks.default_prompts import INTEGER_INDICES, LETTER_INDICES
+from lighteval.tasks.templates.translation_literals import TranslationLiterals
+
+
+ChoicePrefix = Literal["Letters", "NativeLetters", "Numbers"]
+
+
+@dataclass
```

Comment:
What is this file for?

Reply:
It's for the for the normalization primitives, so that it's easy to switch between cf/mcf/hybrid formulations